Tasks/NuGetCommand/nugetcommandmain.ts

```diff
@@ -7,15 +7,18 @@ import * as nugetPack from './nugetpack';
 import * as nugetCustom from './nugetcustom';
 import nuGetGetter = require("nuget-task-common/NuGetToolGetter");
 
+const NUGET_EXE_CUSTOM_LOCATION: string = "NuGetExeCustomLocation";
+
 async function main(): Promise<void> {
     tl.setResourcePath(path.join(__dirname, "task.json"));
 
     // Getting NuGet
     tl.debug('Getting NuGet');
     let nuGetPath: string = undefined;
     try {
-        nuGetPath = process.env[nuGetGetter.NUGET_EXE_TOOL_PATH_ENV_VAR];
+        nuGetPath = process.env[nuGetGetter.NUGET_EXE_TOOL_PATH_ENV_VAR] || process.env[NUGET_EXE_CUSTOM_LOCATION];
```

Comment:
While debugging this it was difficult to modify this value as you need to set this directly in the agent. Could we also get this from a build task variable? Or am I misunderstanding the use of nuGetPath here?